### PR TITLE
[MAPS_API_KEY] Use distinct keys from GCP projects corresponding to the  environment

### PR DESCRIPTION
**Pull request by shifucun**
Set the maps api key as environment variable instead of hard coded.

### CODE REVIEW

**Review by beets — COMMENTED** [review]
Should we also use secret manager for staging and prod? I don't think we should put keys in github (I've received warnings about other keys I've owned in the past).

**Comment by shifucun** [comment]
> Should we also use secret manager for staging and prod? I don't think we should put keys in github (I've received warnings about other keys I've owned in the past).

Put all api key in secret manager now

File: deployment/website.yaml.tmpl [hunk 1]
```diff
@@ -70,4 +70,6 @@ spec:
             - name: FLASK_ENV
               value: <FLASK_ENV>
             - name: GCS_BUCKET
-              value: <GCS_BUCKET>
\ No newline at end of file
+              value: <GCS_BUCKET>
+            - name: PROJECT
```
Comment (beets) [inline deployment/website.yaml.tmpl:74]: project is very generic -- do you mean GCP project here?
Maybe we can add comments to the yaml.tmpl files?

**Review by beets — COMMENTED** [review]
thanks for moving the keys!

File: deployment/website.yaml.tmpl (hunk 1 above)
Comment (shifucun) [reply deployment/website.yaml.tmpl]: renamed to SECRET_PROJECT. I like this suggestion as I once have to remember what project this is.

Comment (beets) [reply deployment/website.yaml.tmpl]: 👍

**Review by beets — APPROVED** [review]

**Pull request merged**